src/cache/edmf_precomputed_quantities.jl

```diff
@@ -132,8 +132,9 @@ function set_edmf_precomputed_quantities!(Y, p, ᶠuₕ³, t)
         sgsʲs_ρaq_tot_int_val =
             Fields.field_values(Fields.level(Y.c.sgsʲs.:($j).ρaq_tot, 1))
 
-        @. sgsʲs_ρa_int_val =
-            $(FT(0.1)) * TD.air_density(thermo_params, ᶜtsʲ_int_val)
+        tcp = CAP.turbconv_params(params)
```

Comment:
```suggestion
        turbconv_params = CAP.turbconv_params(params)
```
(Just to be consistent as we are calling this `turbconv_params` everywhere else in the code)